### PR TITLE
added newapp

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [X ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/saas-cloud/microsoft-exchange-tracelogs.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
+---
+id: microsoft-exchange-tracelogs
+title: Sumo Logic App for Microsoft Exchange Tracelogs
+sidebar_label: Microsoft Exchange Tracelogs
+description: This Sumo Logic App for Microsoft Exchange Tracelogs provides visibility into Delivered, Failed, Quarantined, Pending, and Spam messages.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/saas-cloud/istio.png')} alt="Thumbnail icon" width="50"/>
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/microsoft-exchange-tracelogs.md:10]: wrong logo

File: docs/integrations/saas-cloud/microsoft-exchange-tracelogs.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+The MessageTrace API offers insight into the handling of emails that have gone through Office 365 for an organization. The Sumo Logic Microsoft Exchange Trace Logs App provides information on the delivery status of messages including Delivered, Failed, Quarantined, Pending and Spam. The app dashboard analyzes message size, sender and receiver locations, and provides threat analysis by utilizing the CrowdStrike threat detection feed.
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/microsoft-exchange-tracelogs.md:12]:
```suggestion
The MessageTrace API offers insight into the handling of emails that have gone through Office 365 for an organization. The Sumo Logic Microsoft Exchange Trace Logs App provides information on the delivery status of messages including Delivered, Failed, Quarantined, Pending, and Spam. The app dashboard analyzes message size, sender and receiver locations, and provides threat analysis by utilizing the CrowdStrike threat detection feed.
```

File: docs/integrations/saas-cloud/microsoft-exchange-tracelogs.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+| where country_name matches "{{country}}"
+| timeslice 1d
+| count by _timeslice, status
+| fillmissing timeslice, values all in status
+| transpose row _timeslice column status
+```
+
+## Collecting logs from Microsoft Exchange Tracelogs App
+
+This section provides instructions for setting up [Cloud-to-Cloud-Integration for Microsoft Exchange Tracelogs App](https://help.sumologic.com/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-tracelogs/) to create the source and use the same source category while installing the app.
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/microsoft-exchange-tracelogs.md:94]:
```suggestion
This section provides instructions for setting up [Cloud-to-Cloud-Integration for Microsoft Exchange Tracelogs App](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-tracelogs) to create the source and use the same source category while installing the app.
```

File: docs/integrations/saas-cloud/microsoft-exchange-tracelogs.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+## Installing the Gmail Trace Logs app
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/microsoft-exchange-tracelogs.md:96]: Microsoft, not Gmail

**Review by kimsauce — CHANGES_REQUESTED** [review]
Comments re: logo, headers, and formatting

File: docs/integrations/saas-cloud/microsoft-exchange-tracelogs.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+| where organization matches "{{organization}}"
+| lookup latitude, longitude, country_code, country_name, region, city, postal_code from geo://location on ip = fromIP
+| where country_name matches "{{country}}"
+| timeslice 1d
+| count by _timeslice, status
+| fillmissing timeslice, values all in status
+| transpose row _timeslice column status
+```
+
+## Collecting logs from Microsoft Exchange Tracelogs App
```
Comment (parthiv-sumo) [inline docs/integrations/saas-cloud/microsoft-exchange-tracelogs.md:92]: Collecting logs from Microsoft Exchange Trace Logs App

File: docs/integrations/saas-cloud/microsoft-exchange-tracelogs.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+      * Select **Enter a Custom Data Filter** and type in a custom source category that starts with an underscore. For Example: `_sourceCategory=MyCategory`.
+    3. **Advanced**. Select the **Location in Library** (the default is the **Personal** folder in the library), or click **New Folder** to add a new folder.
+4. Click **Add to Library**.
+
+Once an app is installed, it will appear in your **Personal** folder, or other folder that you specified. You can share it with your organization.
+
+The panels will begin to fill automatically. It's worth noting that each panel gradually fills with data that matches the time range query and has been received since the panel was created. The results will not be available right away, but with some patience, you will be able to view full graphs and maps.
+
+
+## Viewing Dashboards
```
Comment (parthiv-sumo) [inline docs/integrations/saas-cloud/microsoft-exchange-tracelogs.md:119]: Viewing Microsoft Exchange Trace Logs Dashboards

**Review by parthiv-sumo — CHANGES_REQUESTED** [review]
left minor comments

File: docs/integrations/saas-cloud/microsoft-exchange-tracelogs.md [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/saas-cloud/istio.png')} alt="Thumbnail icon" width="50"/>
+
+The MessageTrace API offers insight into the handling of emails that have gone through Office 365 for an organization. The Sumo Logic Microsoft Exchange Trace Logs App provides information on the delivery status of messages including Delivered, Failed, Quarantined, Pending and Spam. The app dashboard analyzes message size, sender and receiver locations, and provides threat analysis by utilizing the CrowdStrike threat detection feed.
+
+## Log types
+
+The Sumo Logic App for Microsoft Exchange Tracelogs uses [MessageTrace reports](https://learn.microsoft.com/en-us/previous-versions/office/developer/o365-enterprise-developers/jj984335(v=office.15). To learn about the Microsoft Exchange Trace logs fields, see [Fields](https://learn.microsoft.com/en-us/previous-versions/office/developer/o365-enterprise-developers/jj984335(v=office.15)#fields) page.
```
[the remark this answers is not in the document]
Comment (JainM6) [reply docs/integrations/saas-cloud/microsoft-exchange-tracelogs.md:16]: Thanks Kim for the suggestion. However, I believe for the better understanding “page” word usage is appropriate here. I am asking user to see the Fields page and read the information.

File: docs/integrations/saas-cloud/microsoft-exchange-tracelogs.md (hunk 4 above)
Comment (JainM6) [reply docs/integrations/saas-cloud/microsoft-exchange-tracelogs.md:96]: Thank you so much for pointing out this.

File: docs/integrations/saas-cloud/microsoft-exchange-tracelogs.md (hunk 6 above)
Comment (JainM6) [reply docs/integrations/saas-cloud/microsoft-exchange-tracelogs.md:119]: Thank you for the suggestion. However, I believe we don’t have to mention the title again. “Viewing Dashboards” looks appropriate.

File: docs/integrations/saas-cloud/microsoft-exchange-tracelogs.md (hunk 1 above)
Comment (JainM6) [reply docs/integrations/saas-cloud/microsoft-exchange-tracelogs.md]: Thank you so much for this. Removed it as we don’t have the logo not on. C2C. Will update in both once I get it.

File: docs/integrations/saas-cloud/microsoft-exchange-tracelogs.md (hunk 5 above)
Comment (JainM6) [reply docs/integrations/saas-cloud/microsoft-exchange-tracelogs.md:91]: What’s the change you suggested?

Comment (JainM6) [reply docs/integrations/saas-cloud/microsoft-exchange-tracelogs.md]: Got it. Fixed. Thanks a lot.

**Comment by JainM6** [comment]
> Comments re: incorrect logo, headers, and formatting



> Comments re: incorrect logo, headers, and formatting

@kimsauce : Thank you for the suggestions, I have fixed all except the “page” word. Closing this comment.

**Review by nmohantyctr — APPROVED** [review]

**Pull request merged**